src/modules/ledger/dto/ledger.dto.ts

```diff
@@ -0,0 +1,31 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+
+export class LedgerRequestParamsDto {
+  @ApiProperty({
+    example: ['arg1', 'arg2'],
+    description: 'Arguments to Ledger request',
+    required: true,
+  })
+  @IsNotEmpty()
+  @IsString()
```

Comment:
Should this not be `@IsString({ each: true })` since this is an array?

Reply:
yes, you are right